.circleci/cache-version.txt

```diff
@@ -1,3 +1,3 @@
 # Bump this version to force CI to re-create the cache from scratch.
 
-09-30-24-windows-patch-server-2022
```

Comment:
we might want to bump this to something fresh to make sure everything is installed properly